### PR TITLE
Q209 Add minimumSizeSubarraySum.py

### PR DESCRIPTION
**Pull request by peinanteng**

### CODE REVIEW

File: minimumSizeSubarraySum.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        if sum(nums) < s:
+            return 0
+        res = len(nums)
+        i, j = 0, 0
+        while j < len(nums) and sum(nums[i : j + 1]) < s:
+            j += 1
+        res = min(res, j - i + 1)
+        while j < len(nums):
+            i = j - res + 1
+            if sum(nums[i : j + 1]) >= s:
```
Comment (nkukarl) [inline minimumSizeSubarraySum.py]: Do you really need this `if` given that you are doing the same check in the following `while`?

File: minimumSizeSubarraySum.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+class Solution(object):
```
Comment (nkukarl) [inline minimumSizeSubarraySum.py]: Without any comments, the code is really hard to understand.
Please add comments.

File: minimumSizeSubarraySum.py (hunk 1 above)
Comment (nkukarl) [reply minimumSizeSubarraySum.py]: You have not addressed previous comments/

Comment (peinanteng) [reply minimumSizeSubarraySum.py]: If is needed, because I need to update the res.

**Pull request merged**